server/services/conditionsService.ts

```diff
@@ -75,17 +86,19 @@ export class ConditionsService {
     }
   }
 
-  getStandardConditions() {
+  getStandardConditionsByVersion(standardConditionsVersion: StandardConditionsVersion) {
+    if (standardConditionsVersion == 1) return standardConditionsV1
+    if (standardConditionsVersion == 2) return standardConditionsV2
```

Comment:
this logic should live in ConditionService like `getAdditionalConditionsConfig`.

This class shouldn't reference specific condition versions

Reply:
also might be good to make the defaulting explicit - only if there are no set of conditions present for a version, easy to miss if we add a version 3